app/javascript/src/components/GameLog.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="border border-black rounded p-2 m-2">
+    
+    <div>
```

Comment:
Is this `div` necessary? Can we remove it?

Reply:
Nope, it can be removed.